packages/core/src/queries/application.ts

```diff
@@ -49,11 +48,6 @@ export const deleteApplicationById = async (id: string) => {
     where id=${id}
   `);
   if (rowCount < 1) {
-    throw new RequestError({
-      code: 'entity.not_exists_with_id',
-      name: Applications.tableSingular,
-      id,
-      status: 404,
-    });
+    throw new SlonikError('Resource not found');
```

Comment:
will it be better to define a new error class say `SlonikDeletionError` which extends `SlonikError` for accurate reference? we can add an issue and do it later

Reply:
LOG-1328